src/Resources/views/CRUD/display_boolean.html.twig

```diff
@@ -9,7 +9,7 @@ file that was distributed with this source code.
 
 #}
 
-{%- apply spaceless %}
+{%- apply trim|raw %}
```

Comment:
so in most cases I just removed the spaceless filter completely. 

Here (and similar for all `display_*.html.twig` cases) this produces extra whitespace within HTML tags then. So `trim|raw` is the closes to `spaceless` I could come up with here.

Any other idea?

Reply:
This is due to `spaceless` doing `trim(preg_replace('/>\s+</', '><', $content ?? ''))`. So its also trimming the string